code/modules/mob/living/living.dm

```diff
@@ -1041,4 +1041,32 @@
 
 /mob/living/onTransitZ(old_z,new_z)
 	..()
-	update_z(new_z)
\ No newline at end of file
+	update_z(new_z)
+
+/mob/living/MouseDrop(mob/over)
+	. = ..()
+	var/mob/living/user = usr
+	if(!istype(over) || !istype(user))
```

Comment:
shouldn't there be some sort of `canUseTopic()` check here?

Reply:
aadded

Reply:
added